systers_portal/templates/meetup/add_meetup.html

```diff
@@ -0,0 +1,42 @@
+{% extends "meetup/base.html" %}
+
+{% block title %}
+  Add meetup to {{ meetup_location.name }}
+{% endblock %}
+
+{% block head %}
+<link  href="/static/css/jquery-ui.min-1.11.4.css" rel="stylesheet" type="text/css"/>
```

Comment:
There's an extra space after "link". Not too relevant, just a cosmetic fix.
Also the next line is indented - it's probably an accident. Again cosmetic fix.